packages/SwingSet/test/vat-warehouse/test-warehouse.js

```diff
@@ -10,16 +10,26 @@ import { test } from '../../tools/prepare-test-env-ava.js';
 import fs from 'fs';
 import tmp from 'tmp';
 import { initSwingStore } from '@agoric/swing-store';
-import { loadBasedir, buildVatController } from '../../src/index.js';
+import { buildVatController } from '../../src/index.js';
```

Comment:
what motivated this change?

Reply:
I added a few new `vat-*.js` files to the directory, for the new test, but `loadBasedir()` loads everything it can find, so the old test loaded them too, which changed the vatIDs it was looking for. The rewritten form makes it insensitive to new files.